compose.py

```
@@ -0,0 +1,305 @@
+                          required=False,
+                          help='Disable console output.')
+    group_qv.add_argument('-v',
+                          '--verbose',
+                          action="store_true",
+                          required=False,
+                          help='Enable verbose output.')
+    parser.add_argument(
+        '--build-name',
+        type=str,
+        required=False,
+        help=
+        'Build name. Default is "tritonserver_build".'
+    )
+    parser.add_argument(
+        '--build-dir',
+        type=str,
+        required=True,
+        help=
+        'Build directory. All repo clones and builds will be performed in this directory.'
+    )
+    parser.add_argument(
+        '--version',
+        type=str,
+        required=False,
+        help=
+        'The Triton version. If not specified defaults to the value in the TRITON_VERSION file.'
+    )
+    parser.add_argument(
+        '--container-version',
+        type=str,
+        required=False,
+        help=
+        'The Triton container version to build. If not specified the container version will be chosen automatically based on --version value.'
+    )
+    parser.add_argument(
+        '--upstream-container-version',
+        type=str,
+        required=False,
+        help=
+        'The upstream container version to use for the build. If not specified the upstream container version will be chosen automatically based on --version value.'
+    )
+    parser.add_argument(
+        '--backend',
+        action='append',
+        required=False,
+        help=
+        'Include specified backend in build as <backend-name>. To have multiple backends built, specify --backend <foo> --backend <bar>'
+    )
+    parser.add_argument(
+        '--repoagent',
+        action='append',
+        required=False,
+        help='Include specified repo agent in build as <repoagent-name>')
+    FLAGS = parser.parse_args()
+
```

Add --enable-gpu as in build.py. For now not supporting cpu-only build so should raise error if --enable-gpu is not specified.